pydocx/openxml/wordprocessing/run.py

```diff
@@ -29,18 +17,19 @@ class Run(XmlModel):
     properties = XmlChild(type=RunProperties)
 
     children = XmlCollection(
-        EmbeddedObject,
-        TabChar,
-        Break,
-        NoBreakHyphen,
-        Text,
-        Drawing,
-        Picture,
-        DeletedText,
-        FootnoteReference,
-        FootnoteReferenceMark,
-        FieldChar,
-        FieldCode,
+        'wordprocessing.EmbeddedObject',
+        'wordprocessing.TabChar',
+        'wordprocessing.Break',
+        'wordprocessing.NoBreakHyphen',
+        'wordprocessing.Text',
+        'wordprocessing.Drawing',
+        'wordprocessing.Picture',
+        'wordprocessing.DeletedText',
+        'wordprocessing.FootnoteReference',
+        'wordprocessing.FootnoteReferenceMark',
+        'wordprocessing.FieldChar',
+        'wordprocessing.FieldCode',
+        'markup_compatibility.AlternateContent',
```

Comment:
I don't know how to solve your problem then. If we want to allow strings (which I am +1 on), then we have to rely on tests. If we don't want strings, then we have to update things in odd places.